Documentation/ColumnsConfig/Type/File/_Snippets/_migration2.diff

```diff
@@ -1,45 +0,0 @@
-
```

Comment:
This code snippet I cannot see when I render locally.

Reply:
This code snippet was removed, therefore it is not displayed anymore?